ampersand-view.js

```diff
@@ -140,16 +139,27 @@ assign(View.prototype, {
     // initialization logic.
     initialize: function () {},
 
-    // **render** is the core function that your view can override, its job is
-    // to populate its element (`this.el`), with the appropriate HTML.
-    render: function () {
+    // **render** is the core function that your view can override. Its job is
+    // to populate its element (`this.el`), with the appropriate HTML. __render
+    // is overriden by any user-defined render.  Each call to `render()` actually
+    // executes `__render()` && `_render()`
+    __render: function () {
         this.renderWithTemplate(this);
+        this._render();
         return this;
     },
 
-    // Remove this view by taking the element out of the DOM, and removing any
-    // applicable events listeners.
-    remove: function () {
+    // called on every `render()` call, regardless if user overrides render()
+    _render: function() {
+        this.rendered = true;
+        return this;
```

Comment:
in retrospect, that would mean that if you override `render` or `remove`, and inside your new fn you make a call to `View.prototype.remove` (~common practice, I'd say), your overridden fn would call `_afterRemove` **&&** the proto fn would call `_afterRemove`.  all heck would break loose.

stepping back further, using the same logic from above, this PR suffers that even `change:rendered` gets triggered twice.  that's a bug needing to be addressed.